stdlib/public/runtime/SwiftObject.mm

```diff
@@ -464,6 +464,8 @@ - (BOOL)isNSValue__      { return NO; }
 static uintptr_t const objectPointerIsObjCBit = 0x4000000000000000ULL;
 #elif defined(__arm64__)
 static uintptr_t const objectPointerIsObjCBit = 0x4000000000000000ULL;
+#elif defined(__s390x__)
+static uintptr_t const objectPointerIsObjCBit = 0x4000000000000000ULL;
```

Comment:
This change shouldn't be needed, since this code is behind `#if SWIFT_OBJC_INTEROP`.